.github/workflows/publish-deps.yml

```diff
@@ -0,0 +1,55 @@
+name:                                               Publish Dependencies to Docker hub
+
+on:
+  push:
+    tags:
+      - v*
+    paths-ignore:
+      - 'README.md'
+      - diagrams/
+  schedule:                                         # Nightly build
+    - cron:                                         '0 0 * * *'
```

Comment:
I don't think this needs to be done nightly. Rather, we should do it weekly, but even that might be too often. It's set to nightly right now though so we can get an "official" image published quickly

Reply:
Logged here as well: https://github.com/paritytech/parity-bridges-common/issues/627